### ISSUE TITLE
FMVs still broken

### ISSUE THREAD

**Issue by drdaxxy** [Kaldaien/TZF#9]
Figured I should create a new issue for this.

BinkClose appears to be called long before the video ends - right as it begins, even, I think.
I haven't found any Bink function that gets called only after playing a video, either.

That said, the main video loop seems to be in the (Zestiria) function at 0x00622C00. We could inject code at the end of that.
